src/content/docs/apm/agents/php-agent/installation/php-agent-installation-arm64.mdx

```diff
@@ -34,7 +34,7 @@ ARM64 is only supported for Amazon Linux 2 and CentOS Linux 8 (including use wit
     sudo amazon-linux-extras install -y  php8.0
```

Comment:
Needs this format

title: 'Insert a title here' 
summary: 'Insert a brief description of the feature' 
releaseDate: 'yyyy-mm-dd' 
learnMoreLink (optional): 'https://example.com' 
getStartedLink (optional): 'https://example.com'